pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [build-system]
-requires = ["poetry"]
-build-backend = "poetry.masonry.api"
+requires = ["poetry_core>=1.0.0"]
```

Comment:
@azawlocki just out of curiosity - where do thse changes come from?

Reply:
From the remarks at the end of https://python-poetry.org/docs/pyproject/ (I was reading that file looking for a way to add `goth` as an extra/optional dependency).